### ISSUE TITLE
requestOptions are broken with custom instance

### ISSUE THREAD

**Issue by aizerin** [anymaniax/orval#766]
## What are the steps to reproduce this issue?

1. download react-query basic example
2. add `requestOptions` to some operation and use custom instance

```javascript
export default defineConfig({
  petstore: {
    output: {
      mode: 'split',
      target: 'src/api/endpoints/petstoreFromFileSpecWithTransformer.ts',
      schemas: 'src/api/model',
      client: 'react-query',
      override: {
        mutator: {
          path: './src/api/mutator/custom-instance.ts',
          name: 'customInstance',
        },
        operations: {
          listPets: {
            requestOptions: { foo: 'bar' },
          },
        },
      },
    },
    input: {
      target: './petstore.yaml',
    },
  },
});
```
4. run generator

## What happens?

Orval will generate invalid code. Request options are outside of config object.

```typescript
export const listPets = (
    params?: ListPetsParams,
 signal?: AbortSignal
) => {
      return customInstance<Pets>(
      {url: `/pets`, method: 'get',
        params, signal
    },
       foo: 'bar', );
    }
```

## What were you expecting to happen?

request options should be inside of config object

```typescript
export const listPets = (
    params?: ListPetsParams,
 signal?: AbortSignal
) => {
      return customInstance<Pets>(
      {url: `/pets`, method: 'get',
        params, signal,  foo: 'bar'
    } );
    }
```

## Any logs, error output, etc?

…

## Any other comments?

…

## What versions are you using?

**Operating System:** MacOS
**Package Version:** 6.11.1

**Comment by anymaniax** [comment]
Fixed will be in the next release

**Comment by anymaniax** [comment]
Released with `6.12.0`
